scripts/new_project.kts

```diff
@@ -1,25 +1,68 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
+
+import kotlinx.coroutines.*
+import java.io.File
+import kotlin.coroutines.coroutineContext
+
 object NewProject {
 
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
     private const val ARGUMENT_DELIMITER = "="
+    private const val COROUTINE_FOLDER_NAME = "CoroutineTemplate"
+    private const val COROUTINE_PACKAGE_THIRD_SEG = "coroutine"
 
     private var appName = ""
     private var packageName = ""
+    private var appNameWithoutSpace = ""
 
-    fun generate(args: Array<String>) {
+    fun generate(args: Array<String>) = runBlocking {
         handleArguments(args)
+
+        showMessage("=> 🐢 Initializing new project...")
+        copyFiles(to = appNameWithoutSpace)
+        /*
+         * Set gradlew file as executable
+         * Copying file from one folder to another lost it's previous execution right.
+         * Need to set gradlew as executable to run
+         */
+        File("$appNameWithoutSpace/gradlew")?.setExecutable(true)
     }
 
     private fun handleArguments(args: Array<String>) {
         args.forEach {
             val (key, value) = it.split(ARGUMENT_DELIMITER)
             when (key) {
-                KEY_APP_NAME -> appName = value
+                KEY_APP_NAME -> {
+                    appName = value
+                    appNameWithoutSpace = appName.trim().replace(" ", "")
+                }
                 KEY_PACKAGE_NAME -> packageName = value
             }
         }
     }
+
+    private fun showMessage(message: String?) {
+        runBlocking {
+            println("\n${message?.trimIndent()}\n")
+        }
+    }
+
+    private suspend fun copyFiles(to: String) {
+        withContext(coroutineContext) {
+            async(Dispatchers.IO) {
+                val target = File(to)
+                val file = File("../$COROUTINE_FOLDER_NAME")
+                file.copyRecursively(target, true) { file, exception ->
+                    showMessage(exception?.message)
+                    return@copyRecursively OnErrorAction.TERMINATE
+                }
+            }
+        }
+    }
 }
 
-NewProject.generate(args)
+NewProject.generate(args)
```

Comment:
Please keep the new line at the end of the file 🚀

Reply:
@kaungkhantsoe You can resolve this, by enabling "Ensure every saved file ends with a line break:

<img width="977" alt="Screen Shot 2565-05-11 at 10 21 55" src="https://user-images.githubusercontent.com/18277915/167762571-3f9a8cc2-607b-439e-9c5c-83bb0da209ca.png">